crates/matrix-sdk/src/docs/sync_running.md

```diff
@@ -0,0 +1 @@
+This method requires a running sync to complete.
```

Comment:
I think some more detail would be good, like ", that is one of [list possible API calls]". Also this should definitely be its own paragraph (which means you need to add a leading newline), or maybe even have its own small heading (in which case you don't need that leading newline).

Reply:
How about now?